main.js

```diff
@@ -0,0 +1,105 @@
+/* eslint no-restricted-syntax: "off", curly: "off" */
+/* eslint no-use-before-define: "off", curly: "off" */
+/* eslint no-unused-vars: "off", curly: "off" */
+/* eslint func-names: "off", curly: "off" */
```

Comment:
- [ ] **(OPTIONAL)** Why don't you try using ES6 arrow functions instead of disabling this?